packages/dashboard/src/index.html

```diff
@@ -10,9 +10,9 @@
     <script type="module" src="/build/iot-app-kit-dashboard.esm.js"></script>
     
 
-    <script type="module" src="http://localhost:3334/build/iot-app-kit-components.esm.js"></script>
-    <link rel="stylesheet" href="http://localhost:3334/build/iot-app-kit-components.css">
-    <script nomodule src="http://localhost:3334/build/iot-app-kit-components.js"></script>
+    <script type="module" src="http://localhost:3333/build/iot-app-kit-components.esm.js"></script>
```

Comment:
Is this necessary for some reason? If not, can we remove it?

Reply:
From testing with these changes, I don't think these lines are necessary. I've removed them.